docs/pages/router/basics/notation.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Expo Router notation
+description: Learn how to use special filenames and notation to expressively define your app's navigation tree within your project's file structure.
+sidebar_title: Router notation
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+
+## Expo Router notation
+
+Look inside the app folder in a typical Expo Router project, and you'll see a lot more than just some simple file and folder names. What do the parenthesises and brackets mean? Let's learn what the significance of file-based routing notation and how it allows you to define complex navigation patterns.
+
+### Simple names / no notation
+
+<FileTree files={[['app/home.tsx'], ['app/feed/favorites.tsx']]} />
+
+Regular folder and file names without any notation signify _static routes_. Their URL matches exactly as they appear in your file tree. So, a file named **favorites.tsx** inside the **feed** folder will have a URL of `/feed/favorites`.
+
+### Square brackets
+
+<FileTree files={[['app/[userName].tsx'], ['app/products/[productId]/index.tsx']]} />
+
+If you see square brackets in a folder or file name, you are looking at a _dynamic route_. The name of the route includes a parameter that can be used when rendering the page. The parameter could be either in a folder name or a file name. For example, a file named **[userName].tsx** will match `/evanbacon`, `/expo`, or another username. Then you can access that parameter with the `useLocalSearchParams` hook inside the page, using that to load the data for that specific user.
+
+### Parentheses
+
+<FileTree files={[['app/(tabs)/index.tsx'], ['app/(tabs)/settings.tsx']]} />
+
+A folder with its name surrounded in parenthesis indicates a _route group_. These folders are useful for grouping routes together without affecting the URL. For example, a file named **(tabs)/settings.tsx** will have `/settings` for its URL, even though it is not directly in the **app** directory.
```

Comment:
```suggestion
A directory with its name surrounded in parentheses indicates a _route group_. These directories are useful for grouping routes together without affecting the URL. For example, a file named **app/(tabs)/settings.tsx** will have `/settings` for its URL, even though it is not directly in the **app** directory.
```

The (tabs) directory will be initially inside the **app** directory? I think we should use a complete path here just to avoid causing any confusion. Reading last line says otherwise.